test/relayserver/RelayServer.test.ts

```diff
@@ -429,20 +417,79 @@ describe('RelayServer', function () {
     let smartWallet: SupportedSmartWallet;
     let hubInfo: HubInfo;
     let encodedData: string;
+    let smartWalletFactory: SmartWalletFactory;
+    let boltzVerifier: BoltzDeployVerifier;
+    let boltzFactory: BoltzSmartWalletFactory;
+    let minimalBoltzVerifier: MinimalBoltzDeployVerifier;
+    let minimalBoltzFactory: MinimalBoltzSmartWalletFactory;
+
+    async function prepareVerifier<
+      F extends SupportedSmartWalletFactory,
+      V extends SupportedDeployVerifier
+    >(
+      fundedAccount: SignerWithAddress,
+      type: SupportedType
+    ): Promise<{ factory: F; verifier: V }> {
+      const smartWalletTemplate: SupportedSmartWallet = await deployContract(
+        `${type === 'Default' ? '' : type}SmartWallet`
```

Comment:
We use this in other places. Couldn't we create a function for this?
e.g.:
```
const getSmartWalletTemplate = (type) => `${type === 'Default' ? '' : type}SmartWallet`;
```

Reply:
not really, but I updated other parts to reuse an existing function.